### PR TITLE
scan: data source index page with header

### LINKED ISSUE #651: Implement scan data source page with header portion

### PR DESCRIPTION
**Pull request by evilpeach**
Fixes #651 

- [x] add more font size (18px)
- [x] add MockAddress in InfoHL for mocking address (remove later)
- [x] add DataSourceIndexPage to the route
- [x] create DataSourceIndexPage (only header ui)

### CODE REVIEW

File: scan/src/reusable/InfoHL.re [hunk 1]
```diff
@@ -6,7 +6,8 @@ type t =
   | Fee(float)
   | DataSources(list(string))
   | Hash(Hash.t, Css.Types.Color.t)
-  | Address(Address.t, Css.Types.Color.t);
+  | Address(Address.t, Css.Types.Color.t)
+  | MockAddress(Css.Types.Color.t);
```
Comment (sorawit) [inline scan/src/reusable/InfoHL.re]: Don't think we need this to introduce this type.

**Review by sorawit — CHANGES_REQUESTED** [review]

**Comment by sorawit** [comment]
@prin-r Please check if this follows your design

File: scan/src/pages/DataSourceIndexPage.re [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            value="DATA SOURCE"
+            weight=Text.Semibold
+            size=Text.Md
+            nowrap=true
+            color=Colors.grayHeader
+            block=true
+          />
+          <div className=Styles.seperatedLine />
+          <Text
+            value="Last updated 4 hours ago"
```
Comment (prin-r) [inline scan/src/pages/DataSourceIndexPage.re:39]: Use letter-spacing: 0.06em;

File: scan/src/pages/DataSourceIndexPage.re [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+[@react.component]
+let make = (~dataSourceID, ~hashtag: Route.data_source_tab_t) => {
+  <div className=Styles.pageContainer>
+    <Row justify=Row.Between>
+      <Col>
+        <div className=Styles.vFlex>
+          <img src=Images.dataSourceLogo className=Styles.logo />
+          <Text
+            value="DATA SOURCE"
```
Comment (prin-r) [inline scan/src/pages/DataSourceIndexPage.re:28]: Use letter-spacing: 0.06em;

File: scan/src/pages/DataSourceIndexPage.re [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      <Text
+        value="#D253"
+        size=Text.Xxl
+        weight=Text.Bold
+        color=Colors.brightOrange
+        nowrap=true
+        code=true
+      />
+      <HSpacing size=Spacing.md />
+      <Text value="CoinGecko V.2" size=Text.Xxl weight=Text.Bold nowrap=true />
```
Comment (prin-r) [inline scan/src/pages/DataSourceIndexPage.re]: Use line-height: 22px;

File: scan/src/pages/DataSourceIndexPage.re [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+          header="OWNER"
+          info={
+            InfoHL.Address(
+              "band1gfskuezzv9hxgsnpdejyyctwv3pxzmnywps0q9" |> Address.fromBech32,
+              Colors.grayHeader,
+            )
+          }
+        />
+      </Col>
+      <Col size=0.8> <InfoHL info={InfoHL.Fee(1000.)} header="REQUEST FEE" /> </Col>
```
Comment (prin-r) [inline scan/src/pages/DataSourceIndexPage.re:82]: Use font-weight: 500;

File: scan/src/pages/DataSourceIndexPage.re [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+        code=true
+      />
+      <HSpacing size=Spacing.md />
+      <Text value="CoinGecko V.2" size=Text.Xxl weight=Text.Bold nowrap=true />
+    </div>
+    <VSpacing size=Spacing.xl />
+    <Row>
+      <Col size=1.>
+        <InfoHL
+          header="OWNER"
```
Comment (prin-r) [inline scan/src/pages/DataSourceIndexPage.re:73]: Use font-weight: 500;

File: scan/src/reusable/InfoHL.re [hunk 7]
```diff
@@ -66,12 +72,23 @@ let make = (~info, ~header) => {
...
+       //    value={address |> Address.toBech32}
+       //    size=Text.Lg
+       //    weight=Text.Semibold
+       //    color=textColor
+       //    code=true
+       //  />
+       <div className=Styles.vFlex>
+         <Text value="band" size=Text.Lg weight=Text.Bold color=textColor code=true />
+         <Text
+           value="17rprjgtj0krfw3wyl9creueej6ca9dc4dgxv6e"
```
Comment (prin-r) [inline scan/src/reusable/InfoHL.re:100]: Use letter-spacing: 0.02em;

File: scan/src/reusable/InfoHL.re [hunk 8]
```diff
@@ -45,7 +45,13 @@ let make = (~info, ~header) => {
          weight=Text.Bold
        />
      | Fee(fee) =>
-       <Text value={(fee |> Format.fPretty) ++ " BAND"} size=Text.Lg weight=Text.Bold />
+       <div className=Styles.vFlex>
+         <Text value={fee |> Format.fPretty} size=Text.Lg weight=Text.Bold code=true />
+         <HSpacing size=Spacing.md />
+         <Text value="BAND" size=Text.Lg weight=Text.Regular code=true />
```
Comment (prin-r) [inline scan/src/reusable/InfoHL.re]: Use letter-spacing: 0.02em;

File: scan/src/reusable/InfoHL.re [hunk 9]
```diff
@@ -45,7 +45,13 @@ let make = (~info, ~header) => {
...
+         <HSpacing size=Spacing.xs />
+         <Text value="($0.3)" size=Text.Lg weight=Text.Regular code=true />
```
Comment (prin-r) [inline scan/src/reusable/InfoHL.re]: Use letter-spacing: 0.02em;

**Review by taobun — APPROVED** [review]

**Review by sorawit — APPROVED** [review]
Looks great!

**Pull request merged**